src/main/java/uk/gov/hmcts/reform/sendletter/model/out/StaleLetterResponse.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.hmcts.reform.sendletter.model.out;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+public class StaleLetterResponse {
+
+    @JsonProperty("stale_letters")
+    public final List<StaleLetter> staleLetters;
```

Comment:
an extra `count` field would be a nice touch imho

Reply:
why not?

Reply:
done